### ISSUE TITLE
how to update from version1 to version2 in tensorflow

### ISSUE THREAD

**Issue by messi313** [tflearn/tflearn#1144]
I have NN model in tesorflow1.14 but i want to update as per tensorflow2 but how can i do i am facing problem. I read tensorflow website but not useful or get idea for my code. 

        # Build neural network
net = tflearn.input_data(shape=[None, len(train_x[0])])
net = tflearn.fully_connected(net, 8)
net = tflearn.fully_connected(net, 8)
net = tflearn.fully_connected(net, len(train_y[0]), activation='softmax')
net = tflearn.regression(net)
# Define model and setup tensorboard
model = tflearn.DNN(net, tensorboard_dir='tflearn_logs')
# Start training (apply gradient descent algorithm)
 model.fit(train_x, train_y, n_epoch=100, batch_size=8, show_metric=True)
 model.save('my_drive/AI_values/model/model.ckpt')
